File: ProjetTripAdvisor/DAL/Models/Review.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
    public class Review
    {
        public int ReviewId { get; set; }
        public int Note { get; set; }
        public int UserId { get; set; }
        public string Date { get; set; }
        public int ServiceId { get; set; }
        public string Text { get; set; }

        public Review()
        {
            ReviewId = 0;
            Note = 0;
            UserId = 0;
            Date = " ";
            ServiceId = 0;
            Text = " ";
        }

        public override string ToString()
        {
            return this.ReviewId + " " + this.Note + " " + this.Text;
        }
    }
}

File: frontend/src/review/Review.js
//import { useState, useEffect } from 'react';
import './Review.css';
import 'bootstrap/dist/css/bootstrap.css';


    function Review({id,userid,serviceid,note,text,date}) {
        return (
            <div className="row">
                <div className="col-1">{id}</div>
                <div className="col-1">{userid}</div>
                <div className="col-1">{serviceid}</div>
                <div className="col-1">{note}</div>
                <div className="col-2">{text}</div>
                <div className="col-2">{date}</div>
            </div>
        );
    }
    export default Review;